modules/webstat/lib/webstat_engine.py

```diff
@@ -1886,15 +1886,18 @@ def create_custom_summary_graph(data, path, title):
     """
     # If no input, we don't bother about anything
     if len(data) == 0:
-        return
+        return False
     os.environ['HOME'] = CFG_TMPDIR
 
     try:
         import matplotlib
         matplotlib.use('Agg')
         import matplotlib.pyplot as plt
-    except ImportError:
-        return
+    except:
+        from invenio.errorlib import register_exception
+        register_exception()
+
```

Comment:
Sorry you are right.... It was what I was thinking to do... spread attention is not always a gain of productivity it seems. 

I have already fix it I push it in 5 secs :) !